src/Main.java

```diff
@@ -1,5 +1,55 @@
+import java.time.LocalDate;
+
 public class Main {
+    public static void printALeapYearOrNot(int year) {
+        int fourthYear = year % 4;
+        int hundredthYear = year % 100;
+        int fourHundredthYear = year % 400;
+        if (year > 1584 && fourthYear == 0 && hundredthYear != 0 || year > 1584 && fourthYear == 0 && fourHundredthYear == 0) {
```

Comment:
```suggestion
        if (year > 1584 && fourthYear == 0 && hundredthYear != 0 || fourHundredthYear == 0) {
```
вот такое условие должно быть